README.md

```diff
@@ -41,18 +41,41 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 ## Architecture
 
-| src
-| -- `assets`
-| -- `components`
-| -- `layouts`
-| -- `pages`
-index.tsc
+- `public`
```

Comment:
Is it possible to remove `logo192`from l11 `manifest.json`
<img width="1680" alt="Capture d’écran 2022-01-29 à 23 32 04" src="https://user-images.githubusercontent.com/59573542/151679680-c4a7517f-920a-49c0-b473-1cb497e11fce.png">
?

Reply:
No, these are files for favicon display on devices :
- Manifest.json is used for choose which file is adapted for a specific device by its filepath ;
- logo192 is the favicon image file for android if I remember 🤔